webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'inline-source-map',
+  entry: path.resolve(__dirname, 'src', 'index.js'),
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: '[name].bundle.js',
+    clean: true,
+  },
+  devServer: {
+    static: './dist',
+    watchFiles: ['./src/*'],
+    open: true,
+    hot: true,
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+      title: 'Output Management',
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```

Comment:
Well done brother. Good Job, I am suggesting we will need a rule to load images from our assest folder